### PR TITLE
added option to separate first name and last name in the export

### PR DESCRIPTION
**Pull request by Upellift99**

### CODE REVIEW

File: php/bookings/SeatregBookingsXlsx.php [hunk 1]
```diff
@@ -59,18 +59,34 @@ private function setupXlsx() {
...
+                esc_html__('Name', 'seatreg') => 'string',
+                esc_html__('Email', 'seatreg') => 'string',
+                esc_html__('Registration date', 'seatreg') => 'string',
+                esc_html__('Calendar date', 'seatreg') => 'string',
+                esc_html__('Booking id', 'seatreg') => 'string',
+                esc_html__('Booking status', 'seatreg') => 'string',
+                esc_html__('Booking approval date', 'seatreg') => 'string',
+                esc_html__('Payment status', 'seatreg') => 'string',
+            );
+        }
```
Comment (SiimKirjanen) [inline php/bookings/SeatregBookingsXlsx.php:89]: Lot of code duplication. Do you ideas how to remove them?
Maybe something like 
 ```
$header = array(
            $placeNumberText => 'string',
            esc_html__('Room name', 'seatreg') => 'string',
         );
         
         if ( $this->_separateFirstandLastName ) {
            array_push(
                $header,
                esc_html__('First name', 'seatreg') => 'string',
                esc_html__('Last name', 'seatreg') => 'string',
            );
        } else {
            array_push(
                $header,
                esc_html__('Name', 'seatreg') => 'string',
            );
        }
        
        array_push(
            $header,
            esc_html__('Email', 'seatreg') => 'string',
            esc_html__('Registration date', 'seatreg') => 'string',
            esc_html__('Calendar date', 'seatreg') => 'string',
            esc_html__('Booking id', 'seatreg') => 'string',
            esc_html__('Booking status', 'seatreg') => 'string',
            esc_html__('Booking approval date', 'seatreg') => 'string',
            esc_html__('Payment status', 'seatreg') => 'string',
        );
```

File: php/bookings/SeatregBookingsXlsx.php [hunk 2]
```diff
@@ -79,16 +95,30 @@ public function printXlsx() {
...
+                    esc_html($registration->seat_nr),
+                    esc_html($registration->room_name),
+                    esc_html($registration->first_name) . ' ' . esc_html($registration->last_name),
+                    esc_html($registration->email),
+                    $bookingDate,
+                    esc_html($this->_calendarDate),
+                    $registration->booking_id,
+                    $status
+                );
+            }
```
Comment (SiimKirjanen) [inline php/bookings/SeatregBookingsXlsx.php:121]: Duplication here also

Comment (Upellift99) [reply php/bookings/SeatregBookingsXlsx.php]: you're right !

**Review by SiimKirjanen — APPROVED** [review]
Looks good to me. Thank you again.

**Pull request merged**